core/src/tangram.cpp

```diff
@@ -229,7 +229,7 @@ void applySceneUpdates() {
 
```

Comment:
It stays valid in a sequential execution, but the address of the reference is copied in a lambda for deferred execution, where it potentially becomes invalid.